provider/terraform/custom_code.rb

```diff
@@ -98,11 +105,22 @@ def config_test
         ))
       end
 
+      def ignore_read_extras
+        @ignore_read_extra ||= []
+        ignore_read_extra.map { |i| "\"#{i}\"" }
```

Comment:
this should be go_literal(i)

except at this point we actually should probably have go_literal able to take an array, and then it would output the []string{"thing", "other thing"} that we want

Reply:
Changed this to go_literal, but didn't add the list functionality since I'm putting this map together with another one later.

Reply:
Sure, but you could just concat the two lists and then call go_literal on that, right?

Reply:
Done. Empty arrays are assumed to be string arrays.